<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StringCopy</class>
 <widget class="QWidget" name="StringCopy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>348</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>StringCopy</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_input">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Input">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Input File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Input_OK">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_StrOri">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>381</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Input_out">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Output File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_input_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>180</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="subStr">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
